### PR TITLE
feat(bridge): non-mm wallets

### PR DESCRIPTION
**Pull request by shadab-taiko**

### CODE REVIEW

**Comment by d1onys1us** [comment]
clicking walletconnect and coinbase wallet does not do anything for me on the current preview @shadab-taiko is this expected behavior? it throws some console errors: https://bridge-6rqlqfol1-taikoxyz.vercel.app/ . let me know if you need them or can reproduce.

File: packages/bridge-ui/vite.config.ts.timestamp-1671142216308.mjs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// vite.config.ts
```
Comment (d1onys1us) [inline packages/bridge-ui/vite.config.ts.timestamp-1671142216308.mjs:1]: should this file be gitignored?

Comment (shadab-taiko) [reply packages/bridge-ui/vite.config.ts.timestamp-1671142216308.mjs:1]: @cyberhorsey I am not sure about this, could you check?

Comment (cyberhorsey) [reply packages/bridge-ui/vite.config.ts.timestamp-1671142216308.mjs:1]: How did you generate this file? Did you run build locally or something? This shouldn't be pushed. I think you can add `vite.config.ts.timestamp-*.mjs` to gitignore.

Comment (shadab-taiko) [reply packages/bridge-ui/vite.config.ts.timestamp-1671142216308.mjs]: added to gitignore

**Review by cyberhorsey — DISMISSED** [review]

**Review by cyberhorsey — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**